packages/cozy-harvest-lib/src/models/cipherUtils.js

```diff
@@ -6,6 +6,12 @@ import logger from '../logger'
 
 import { CipherType, UriMatchType } from 'cozy-keys-lib'
 
+const fieldTypes = {
```

Comment:
Nit: these field types are not exposed through `cozy-keys-lib`?

Reply:
Not yet FieldType